### PR TITLE
S06 ajout offres

### PR DESCRIPTION
**Pull request by JulienChevreauPro**

### CODE REVIEW

**Review by ozen2 — APPROVED** [review]

**Review by axeljdev — APPROVED** [review]

File: server/app/controllers/companyActions.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    res.status(200).json(companies);
+  } catch (err) {
+    next(err);
+  }
+};
+
+// Ready to export the controller functions
+module.exports = {
+  browse,
+  // read,
```
Comment (Yoann-Fortin) [inline server/app/controllers/companyActions.js:16]: Comments are no longer useful, given your level of experience.

File: server/app/controllers/jobActions.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    res.status(200).json(jobs);
+  } catch (err) {
+    next(err);
+  }
+};
+
+// Ready to export the controller functions
+module.exports = {
+  browse,
+  // read,
```
Comment (Yoann-Fortin) [inline server/app/controllers/jobActions.js:16]: See comment above

File: server/app/controllers/offerActions.js [hunk 3]
```diff
@@ -11,28 +11,42 @@ const browse = async (req, res, next) => {
...
-      if (offer == null) {
-        res.sendStatus(404);
-      } else {
-        res.json(offer);
-      }
-    } catch (err) {
-      // Pass any errors to the error-handling middleware
-      next(err);
+  try {
+    // Fetch a specific offer from the database based on the provided ID
```
Comment (Yoann-Fortin) [inline server/app/controllers/offerActions.js:15]: See comment above

File: server/app/controllers/technoActions.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  try {
+    const technos = await tables.techno.readAll();
+
+    res.status(200).json(technos);
+  } catch (err) {
+    next(err);
+  }
+};
+
+// Ready to export the controller functions
```
Comment (Yoann-Fortin) [inline server/app/controllers/technoActions.js:13]: See comment above

File: server/database/models/CompanyRepository.js
Comment (Yoann-Fortin) [inline server/database/models/CompanyRepository.js:1]: Why so much comment code?

File: server/database/models/JobRepository.js
Comment (Yoann-Fortin) [inline server/database/models/JobRepository.js:1]: See comment above

File: server/database/models/TechnoRepository.js
Comment (Yoann-Fortin) [inline server/database/models/TechnoRepository.js:1]: See comment above

File: server/app/services/offerTransaction.js
Comment (Yoann-Fortin) [inline server/app/services/offerTransaction.js:1]: To delete please

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Comment by JulienChevreauPro** [comment]
Removed useless code

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**